### PR TITLE
Code review exercise

### PR DESCRIPTION
**Pull request by kkolade**
This project was executed by myself and another applicant Andrea during the Microverse admission process and I will like it to be reviewed to ascertain its compliance to best coding practices

### CODE REVIEW

**Comment by kkolade** [comment]
There are linter errors in your code please check and fix them before I can go ahead with the review process. This submission will be marked as invalid

**Comment by kkolade** [comment]
I think you should use semantic tags and not just divs as this make your code clean and improves SEO
# Required: 
Please change the div tags to html5 semantic tags

**Comment by Favourezeugwa** [comment]
- You have done a great job by leaving polite suggestions and stating reasons why the student should use semantic tags and not just divs ✔️ ✔️ ✔️

- I think you can as well use the `GitHub feature` to leave a review comment. GitHub features will enable you Approve, Comment or require changes in a PR summary 👍🏽 
[How to use Github features](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

- I believe It is essential to use` inline comments` to highlight so that the student can know exactly what line of code you are referring to. 🤗 
[How to use inline comments](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d)

- I think it is important for you to `add emojis `to your comments because emojis will make your suggestion sound friendly as well 🤗 

[OPTIONAL] 👇🏽 
- After the requested change, you can ask the student what they think about your suggestion. 
For example:
"What do you think about these suggestions above? 💭 "


_Why don't we edit your comments to meet all the requirements I have just stated above??_ 🤔

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @kkolade , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

feelclose

## Highlights 💯 
- Added two review comments ✔️ 
- Used Polite suggestions ✔️
- Stated reasons why a change needs to be implemented ✔️

## Required Changes ♻️
_Check the comment ABOVE the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Using inline comments for your review. Here is a link on how to use Github's inline comments , [Link](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d)
- Using Emoji's to make your comments more friendly
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkolade** [comment]
Hi @kkolade good work so far, you are making a great stride
# Required Changes ♻️ 
*Check the comments above*
# Optional Suggestions 🔶 
*Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.*

**Review by GiuseppeTG — CHANGES_REQUESTED** [review]
Hi @kkolade  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Required Changes ♻️

Please read carefully this section of the project requirements: ["How to give a code review?"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md).

There you should find the instructions on how to do inline comments. Feel free to watch the video and follow it step by step. 🤓 

The inline comments are important because the author of the project can easily understand the context of your comment. They point out exactly what part of the code you are commenting on. 👍 

Kindly add at least one inline comment following the project guideline and your project will be approved 😄 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkolade** [comment]
Hi @GiuseppeTG thanks for the review, please help me out here, because the code is so short and the linters has pointed out all the observable error, I found nothing to comment on, and as per inline comment, is that different from what I have above by commenting multiple lines, Thanks. 🙇‍♂️

**Comment by GiuseppeTG** [comment]
> Hi @GiuseppeTG thanks for the review, please help me out here, because the code is so short and the linters has pointed out all the observable error, I found nothing to comment on, and as per inline comment, is that different from what I have above by commenting multiple lines, Thanks. 🙇‍♂️

Hello @kkolade, try to do an inline comment with the issue about semantic tags that you pointed out. 
Make reference to a line of code where you think a different semantic tag should be used, just as an example 🤓 

You don't need to modify anything in your code. Just leave there the review.

**Comment by kkolade** [comment]
Thanks, will do that

**Comment by GiuseppeTG** [comment]
> Thanks, will do that

Yea, you where not supposed to make changes after your own review. So if you want, introduce an error an do a review about it @kkolade

**Comment by kkolade** [comment]
Inline comment added to line 22 of the HTML code @GiuseppeTG , I believe that is ok. Thanks

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @kkolade 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

nss-nichole


 ## Highlights
Polite and respectful comments :heavy_check_mark: 
Provided reasons for suggested changes :heavy_check_mark: 

 ## Required Changes ♻️

- [ ] Well done for implementing the suggested changes, however, I see a lot of fragmented and single commented reviews instead of one major review as indicated below. Kindly ensure that all your suggested changes and comments, including the summary and inline comments are done in one go. 

### Your review

Screenshot from 2022-11-01 11-11-16
Screenshot from 2022-11-01 11-12-06


### Sample review

Screenshot from 2022-11-01 11-26-24


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — COMMENTED** [review]
Hello @kkolade  👋,

_This is an additional review as a result of the second opinion request._

Unfortunately, I cannot spot any inline comments in this PR.
Please watch again [this video](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) from the lesson on how to give a code review.
Please don't forget to use emojis in the inline comments, and please always explain why you ask for changes.
If you follow closely the example given in the video, definitely you will do a good job.
So don't forget, first of all, to give inline comments from the Files changed section from this PR. 
image

After that, when giving the inline comments, use the Start a review button.
image

In the end, please use the Review changes button to leave the summary comment where you mention to the student that the project is not yet complete.
image


I know you can do it! :muscle:

Keep rocking 🚀!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (kkolade) [inline index.html:166]: Do not leave comments that are not directly relevant to the code maintenance or further action in your code base, this is not a professional best practice

File: index.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+          <input
+            type="email"
+            class="form-control"
+            id="email"
+            aria-describedby="emailHelp"
+            placeholder="Enter your mail"
+            required
+          />
+        </div>
+        <div class="mb-3">
```
Comment (kkolade) [inline index.html:44]: There seem to be a lot of divs in your code base, HTML5 semantic tags will do a better job and your code will appear more professionally looking

**Review by kkolade — COMMENTED** [review]
You've done a good job so far, see the inline comment in your HTML code to view and implement them to remove some code smell.
Good luck

**Comment by kkolade** [comment]
Hi @kkolade, 
Great job so far and you are almost there

so_close_1

# Highlights
- well structure HTML code ✔️ 
- Your CSS is properly cascaded ✔️ 
# Required Changes ♻️ 
- [ ] See how you can replace the multiple div tags in your code with HTML5 semantic tags
- [ ] Give your classes more reflective names
*Check the comment under the review*
# Optional Suggestions 
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @kkolade , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉🎉

happy


## Highlights 💯 
- Inline comments used ✔️ 
- Used GitHub Review features ✔️ 
- Gave reasons WHY a change needs to be implemented ✔️ 
- Used GIFs ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Favourezeugwa) [reply index.html:166]: - Great Job using Inline comments ✔️ 

[OPTIONAL] 👍🏽 
Kindly remember to use emojis in the inline comments just as @nicupop729 advised 🍷

**Pull request merged**